pkg/vendir/config/config.go

```diff
@@ -251,7 +249,7 @@ func (c Config) checkOverlappingPaths() error {
 
 	for i, path := range paths {
 		for i2, path2 := range paths {
-			if i != i2 && strings.Contains(path2, path) {
+			if i != i2 && path == path2 {
```

Comment:
this isnt going to catch case of /foo/bar/blah and /foo/bar overlapping. instead we do something like this i think `strings.HasPrefix(path2+filepath.Separator, path+filepath.Separator)`. this anchors to the beginning of the path and since it adds a separator at the end it will not have false positives like in the issue you filed.

Reply:
yes I see it, will update

Reply:
updated